### PR TITLE
Orchestra lineup

### PR DESCRIPTION
**Pull request by J4ken**

### CODE REVIEW

File: app/controllers/api/v1/lineups_controller.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+class API::V1::LineupsController < ApplicationController
+  include ViewPermissionConcern
+
+  def index
+    render :json => Lineup.all
```
Comment (ljukas) [inline app/controllers/api/v1/lineups_controller.rb]: Could use some excepts, like update_at, created_at and other unneeded information

File: app/controllers/api/v1/lineups_controller.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  def destroy
+    require_admin_permission AdminPermission::LIST_CORTEGE_APPLICATIONS
+    lineup = Lineup.find(params[:id])
+    lineup.destroy
+    head :no_content
+  end
+
+  def get_corteges
+    render :json => Lineup.where(cortege: true)
```
Comment (ljukas) [inline app/controllers/api/v1/lineups_controller.rb]: Same as above

File: app/controllers/api/v1/lineups_controller.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  end
+
+  def get_artists
+    render :json => Lineup.where(orchestra: true)
```
Comment (ljukas) [inline app/controllers/api/v1/lineups_controller.rb]: Same as above

File: db/migrate/20170509160431_rename_cortege_lineup_to_lineup.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class RenameCortegeLineupToLineup < ActiveRecord::Migration[5.0]
```
Comment (ljukas) [inline db/migrate/20170509160431_rename_cortege_lineup_to_lineup.rb]: Empty migration?

File: db/schema.rb [hunk 5]
```diff
@@ -10,14 +10,19 @@
...
-ActiveRecord::Schema.define(version: 20170509072215) do
+ActiveRecord::Schema.define(version: 20170509160431) do
 
   create_table "active_funkis_shift_limits", force: :cascade do |t|
     t.integer  "active_limit", default: 0
     t.datetime "created_at",               null: false
     t.datetime "updated_at",               null: false
   end
 
+  create_table "artist_lineups", force: :cascade do |t|
```
Comment (ljukas) [inline db/schema.rb]: What is this empty table? (no need to remove by migrations, just wondering why its here)

File: db/seeds.rb [hunk 6]
```diff
@@ -16,12 +16,13 @@
     ]],
     ['Frågor & Svar', '/faq'],
     ['Festivalen', '#', children: [
+       ['lineup', '/festival/lineups'],
```
Comment (ljukas) [inline db/seeds.rb]: Stor bokstav, samt menar du artist-lineup?

File: db/seeds.rb [hunk 7]
```diff
@@ -16,12 +16,13 @@
...
        ['Spelschema', '/festival/schedule'],
        ['Karta', '/festival/map', active: false],
        ['Årets öl', '/festival/beer']
     ]],
     ['Kårtege', '#', children: [
-        ['Lineup', '/cortege_lineups'],
+        ['Lineup', '/cortege/lineups'],
```
Comment (ljukas) [inline db/seeds.rb]: Kårtegeordning och inte lineup.

File: spec/controllers/artist_lineup_controller_spec.rb [hunk 8]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (ljukas) [inline spec/controllers/artist_lineup_controller_spec.rb]: Remov empty testfile

File: spec/factories/artist_lineups.rb [hunk 9]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```
Comment (ljukas) [inline spec/factories/artist_lineups.rb]: Remove empty FactoryGirl file

File: spec/models/artist_lineup_spec.rb [hunk 10]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (ljukas) [inline spec/models/artist_lineup_spec.rb]: Remove empty test files.

File: db/schema.rb (hunk 5 above)
Comment (J4ken) [inline db/schema.rb]: shit when generating a model

File: db/migrate/20170509160431_rename_cortege_lineup_to_lineup.rb (hunk 4 above)
Comment (J4ken) [inline db/migrate/20170509160431_rename_cortege_lineup_to_lineup.rb]: removed

File: db/schema.rb (hunk 5 above)
Comment (J4ken) [inline db/schema.rb]: removed

File: db/seeds.rb (hunk 6 above)
Comment (J4ken) [inline db/seeds.rb]: Done

File: db/seeds.rb (hunk 7 above)
Comment (J4ken) [inline db/seeds.rb]: Done

File: spec/controllers/artist_lineup_controller_spec.rb (hunk 8 above)
Comment (J4ken) [inline spec/controllers/artist_lineup_controller_spec.rb]: removed

File: spec/factories/artist_lineups.rb (hunk 9 above)
Comment (J4ken) [inline spec/factories/artist_lineups.rb]: removed

File: spec/models/artist_lineup_spec.rb (hunk 10 above)
Comment (J4ken) [inline spec/models/artist_lineup_spec.rb]: removed

**Pull request merged**